index.js

```diff
@@ -6,6 +6,7 @@ const express = require('express'),
     Models = require('./models.js'),
     { check, validationResult } = require('express-validator');
 
+require('dotenv').config()
```

Comment:
You need this read the `.env` file or the app config variable on heroku

https://www.npmjs.com/package/dotenv